npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -162,6 +162,8 @@ export async function makeWebpackConfig (
     dynamicWebpackConfig,
   )
 
+  delete mergedConfig.output?.chunkFilename
```

Comment:
This is a condition that @ZachJW34 uncovered during his initial spike - I believe that if there was a custom `chunkFilename` it was causing issues with the support file. If he can confirm (or correct me) I'll add a comment to document for the future

Reply:
Some frameworks (like Next.js) change this value. Next.js specifically sets it to `'static/chunks/[name].js'` which means the network request to grab a spec becomes `http://localhost:xxxx/<dev-server-public-path>/static/chunks/spec-<x>.js`. Deleting this normalizes all specs to `http://localhost:xxxx/<dev-server-public-path>/spec-<x>.js` which is required since we need to make a fetch request to extract the source map. There might be other ways but this seemed easiest.

I wouldn't delete the `output` key outright, there are a bunch of options that could affect the compilation (like `module`).

Reply:
Right, gotcha. I think the comment has since been added which clarifies a lot, thanks!